exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceArtifactId.java

```diff
@@ -25,29 +25,34 @@
 
 /**
  * A service artifact identifier. It consists of the runtime id in which the service shall be
- * deployed and the service artifact name. The name of Java artifacts usually contains the three
- * coordinates identifying any Java artifact: groupId, artifactId and version.
+ * deployed, the service artifact name and the artifact version. The name of Java
+ * artifacts usually consists of groupId and artifactId.
```

Comment:
artifacts usually consists of the two Maven coordinates: groupId and artifactId.